course7/homeworks/patrickkostjens/complex/calculator.hh

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "complex.hh"
+
+#include <iostream>
+#include <map>
+#include <stack>
+#include <string>
+
+typedef void* (*plugin)(std::stack<Complex>*);
```

Comment:
Prefer modern ```using``` over old-school ```typedef```

Reply:
Use references instead of pointer, unless you want to create a semantics where passing null pointer means something.

Reply:
Fixed.